### PR TITLE
[MWPW-159265] Callout text in RTL not properly aligned

### PR DESCRIPTION
**Pull request by rohitsahu**
For RTL, make callout texts right aligned

Resolves: [MWPW-159265](https://jira.corp.adobe.com/browse/MWPW-159265)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/ae_ar/drafts/mili/kitchen-sink/merch-card
- After: https://MWPW-159265--milo--rohitsahu.hlx.page/ae_ar/drafts/mili/kitchen-sink/merch-card

### CODE REVIEW

File: libs/features/mas/web-components/src/global.css.js [hunk 1]
```diff
@@ -231,6 +231,10 @@ merch-card [slot='callout-content'] > div > div {
     padding: var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxs);
 }
 
+merch-card [slot='callout-content']:dir(rtl) > div > div > div {
+    text-align: right;
```
Comment (mokimo) [inline libs/features/mas/web-components/src/global.css.js:235]: What about just using `text-align: start;` to begin with?

**Review by mokimo — APPROVED** [review]

File: libs/deps/mas/merch-card.js [hunk 2]
```diff
@@ -1643,6 +1643,10 @@ merch-card [slot='callout-content'] > div > div {
     padding: var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxs);
 }
 
+merch-card [slot='callout-content']:dir(rtl) > div > div > div {
+    text-align: right;
```
Comment (rahulgupta999) [inline libs/deps/mas/merch-card.js:1647]: text-align: start, won't need the specifier for

**Review by yesil — APPROVED** [review]

**Review by npeltier — APPROVED** [review]

**Review by overmyheadandbody — COMMENTED** [review]
Shouldn't the icon also be properly aligned and its spacing mirrored? It is now glued to the text.
Screenshot 2024-09-27 at 17 09 59

**Comment by overmyheadandbody** [comment]
Mobile icon is also not horizontally aligned with the text, regardless of RTL/LTR:
Screenshot 2024-09-27 at 17 14 18

**Review by Axelcureno — APPROVED** [review]

**Comment by rahulgupta999** [comment]
@rohitsahu please take care of PR feedback from @overmyheadandbody in a subsequent PR.

**Comment by rohitsahu** [comment]
> @rohitsahu please take care of PR feedback from @overmyheadandbody in a subsequent PR.

Fixed in https://github.com/adobecom/milo/pull/2982 
cc: @overmyheadandbody

**Pull request merged**